lib/div-alg.gi

```diff
@@ -1388,11 +1388,11 @@ if B=[] and GaloisCyc((1-E(2^m1)^i)*z,A1[4][1][2])=(1-E(2^m1)^i) then
   B[1]:=[F,F1,[E(n)^A1[4][1][3]] ];
   B[2]:=[F,F2,[Norm(F3,F1,(1-E(2^m1)^i))*E(n)^A1[4][2][3]]];
 fi;
-if B=[] and GaloisCyc((1+E(2^m1)^i)*z,A1[4][1][2])=(1+E(2^m1)^i) then
+if B=[] and not(E(2^m1)^i=-1) and GaloisCyc((1+E(2^m1)^i)*z,A1[4][1][2])=(1+E(2^m1)^i) then
```

Comment:
The same is about commit message "fixed bug - DCA returning CycAlgs with zero factor set" should state `DecomposeCyclotomicAlgebra` instead of `DCA`. Also, it's not clear from this formulation, whether the bug WAS that it returns CycAlgs with zero factor set, or this is the new correct behaviour after the fix? What about
```
Fix a bug in DecomposeCyclotomicAlgebra

Before this, for certain inputs it could return cyclotomic algebras with zero factor set.
```